scripts/debug_rom/Makefile

```diff
@@ -0,0 +1,31 @@
+# See LICENSE.SiFive for license details
+# Recursive make is bad, but in this case we're cross compiling which is a
+# pretty unusual use case.
+
+CC = $(RISCV)/bin/riscv64-unknown-elf-gcc
+OBJCOPY = $(RISCV)/bin/riscv64-unknown-elf-objcopy
```

Comment:
IIRC we still build riscv32-unknown-elf-gcc for 32-bit targets?  If so, then this should use XLEN.